### ISSUE TITLE
Need Oracle patch to install 19c on EL9

### ISSUE THREAD

**Issue by djvl3it** [oravirt/ansible-oracle#390]
It appears the installation of 19c on EL9 is not as trivial. I stumbled upon an new issue, this time Oracle specific. The issue appeared in the play "TASK [opitzconsulting.ansible_oracle.oraswdb_install : install_home_db | Install Oracle Database Server] ***". This one failed because of:

"In Oracle Linux 9 / Red Hat Enterprise Linux 9, the pthread library has been integrated into glibc.
Therefore, the linker cannot find the pthread library and linking fails."

Solution: Add library stubs by Patch 35775632

I added a quick and dirty workaround by adding the following to `collections/ansible_collections/opitzconsulting/ansible_oracle/roles/oraswdb_install/tasks/install-home-db.yml` on line 89:
````
    - name: Unarchive patch to stubs dir ORACLE_HOME
      ansible.builtin.unarchive:
        src: "{{ oracle_stage_remote }}/p35775632_190000_Linux-x86-64.zip"
        dest: "{{ oracle_home_db }}/lib/stubs"
        remote_src: yes
      when:
        - not oracle_sw_copy
        - oracle_home_db not in existing_dbhome.stdout_lines
        - ansible_distribution_major_version | int == 9

    - name: Unarchive stubs.tar
      ansible.builtin.unarchive:
        src: "{{ oracle_home_db }}/lib/stubs/stubs.tar"
        dest: "{{ oracle_home_db }}/lib/stubs"
        remote_src: yes
      when:
        - not oracle_sw_copy
        - oracle_home_db not in existing_dbhome.stdout_lines
        - ansible_distribution_major_version | int == 9
````
The installation of this patch is also described in:
"Requirements for Installing Oracle Database/Client 19c (19.19 or higher) on OL9 or RHEL9 64-bit (x86-64) (Doc ID 2982833.1)"

**Comment by Rendanic** [comment]
That's exactly the point when I stopped the implementation of OL9 in `ansible-oracle`for the moment.
I hate to implement a workarround for a broken patch from Oracle.
They were not able to create a patch with all inventory data for OPatch....
We have to install that patch manually - without OPatch.

The next issue:
Could I use this home for OL6-8 when building a Golden-Image from it?

I stopped the implementation due to the issue with the patch and the risk to create an ORACLE_HOME which could not be used in in other Distribtions.
I hope to get a smooth solution from Oracle - not this broken thing.

**Comment by Rendanic** [comment]
The fix for `Bug 33794281  libpthread_nonshared.a is needed for oracle database 23c and 19c` should be included in 19.21+.
This makes the installation much easier, because we only need the option to preinstall RUs before configuration.

The preinstall is possible in Grid-Infrastructure but not in normal Database Homes for the moment.

Solution:
Implement the option to preinstall RDBMS RU before configuration.
